software/known-issues.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Known issues
+---
+
+This document describes a list of known issues and potential recovery steps.
+
+- [Gaps Detected](#gaps-detected)
+    - [Symptoms](#symptoms)
+    - [Reasons for this error](#reasons-for-this-error)
+        - [mismatched core/Horizon configurations](#mismatched-corehorizon-configurations)
+            - [Missing cursors](#missing-cursors)
+            - [mismatch policy between core and Horizon](#mismatch-policy-between-core-and-horizon)
+        - [corrupt database](#corrupt-database)
+    - [Recovery process](#recovery-process)
+        - [Reset everything](#reset-everything)
+        - [Bridging the gap](#bridging-the-gap)
```

Comment:
Just a cosmetic change but can we make sure all sections' names start with capitalized letter here and below?